src/verify.c

```diff
@@ -68,6 +68,7 @@ static const struct option prog_opts[] = {
 	{ "quick", no_argument, 0, 'q' },
 	{ "force", no_argument, 0, 'x' },
 	{ "statedir", required_argument, 0, 'S' },
+	{ "verify-sign", no_argument, 0, 'V' },
```

Comment:
"verify-signatures" would also be okay with me. Can you add a configure option to enable the signature verification code at build time? So, if a distro wants to always verify signatures, they can update packaging files and not have to pass the -V option to swupd.